src/NativeScript/GlobalObject.mm

```diff
@@ -7,45 +7,46 @@
 //
 
 #include "GlobalObject.h"
-#include <string>
```

Comment:
This looks good to me 😄 

Maybe we should reformat all sources like this, but this is for another time.